ofrak_core/ofrak/core/free_space.py

```diff
@@ -450,34 +450,37 @@ async def _find_and_delete_overlapping_children(resource: Resource, freed_range:
         await overlapping_child.resource.save()
 
 
-def _get_fill(freed_range: Range, fill: Optional[bytes]):
-    if not fill:
-        return b"\x00" * freed_range.length()
-    else:
-        diff_len = freed_range.length() - len(fill)
-        if diff_len < 0:
-            raise ValueError("config.fill value cannot be longer than the range to be freed.")
-        return fill + b"\x00" * diff_len
+def _get_patch(freed_range: Range, stub: bytes, fill: bytes) -> bytes:
+    total_fill_length = freed_range.length() - len(stub)
+    remainder = total_fill_length % len(fill)
```

Comment:
This will raise a `ZeroDivisionError` if the user passes in a fill=`b""`. I think we should check for `len(fill) == 0` and either raise a `ValueError` stating that `fill must be at least 1 byte`, or to set fill to `b"\x00"` in that case.

Reply:
Good point -- I've addressed this in the config and added tests.